mathics/core/expression.py

```diff
@@ -6,11 +6,25 @@
 import time
 
 import typing
-from typing import Any, Optional
+from typing import Any, Iterable, Optional
 from itertools import chain
 from bisect import bisect_left
 
 from mathics.core.atoms import from_python, Number, Integer
+
+# FIXME: adjust mathics.core.attributes to uppercase attribute names
```

Comment:
@TiagoCavalcante would love to adjust mathics.core.attributes, but I ran out of time. 

What do you think? If you think this makes it clearer would you adjust the rest of the code? 

Basically when I was reading the code and saw flat, it took me a while to figure where this was set. Had it been uppercase that would have cleared things up.

Reply:
I'm not sure about this, the attributes are constants,

Reply:
In C and other languages all uppercase indicates literal values (in C, C preprocessor literals). Even in Python there is some precedent for this. See https://docs.python.org/3/library/stat.html?module-stat#stat.ST_MODE

The other thing I had considered was to add `_attr` or something like that at the end of an attribute. A name like "flat"  to indicate an  constant attribute flag value is weird. And as I said I had to look around to find out what was up. Even _after_ I saw the import it took me another second to realize what was up. If this were marked a constant or named in fashion that suggests bitmap then things would have been fine.